server/auth/types/basic/routes.ts

```diff
@@ -44,7 +44,17 @@ export class BasicAuthRoutes {
       },
       async (context, request, response) => {
         this.sessionStorageFactory.asScoped(request).clear();
-        return response.renderAnonymousCoreApp();
+        let clearOldVersionCookie: string;
+        if (this.config.cookie.secure) {
+          clearOldVersionCookie = 'security_authentication=; Secure; HttpOnly; Path=/';
```

Comment:
Will it set to blank or remove the key?

![image](https://user-images.githubusercontent.com/63078162/99010621-3044ae80-24ff-11eb-8fe7-4af25e6ae260.png)
I tried set a blank key in chrome dev tool and in this way it would still not able to login.

Reply:
Added max-age and expire to clear the cookie. e.g. `security_authentication=; Max-Age=0; Expires=Thu, 01 Jan 1970 00:00:00 GMT; Secure; HttpOnly; Path=/`